roles/openshift_health_checker/openshift_checks/logging/logging.py

```diff
@@ -49,9 +49,10 @@ def get_pods_for_component(self, execute_module, namespace, logging_component, t
 
         return pods['items'], None
 
-    @staticmethod
-    def not_running_pods(pods):
-        """Returns: list of pods not in a ready and running state"""
+    def not_running_pods(self, pods):
+        """Returns: list of pods not in a ready and running state
+        or an OpenShiftCheckException if no pods have been scheduled."""
+        self.ensure_scheduled_pods(pods)
```

Comment:
So after thinking about this for a bit, while I like the idea of informing users about specific problems with their pods, this just does not feel like the right place to do it. And I think there could be legitimate reasons why a pod might not be working but not actually impacting anything because there's already a replacement, so I wouldn't want the check to fail on any particular pod having problems, just if there are the wrong number of pods actually in service.

WDYT?